plugin/src/main/java/com/iridium/iridiumskyblock/utils/StringUtils.java

```diff
@@ -35,6 +36,16 @@ public static String color(String string) {
                 .collect(Collectors.toList());
     }
 
+    /**
+     * Capitalizes the provided string.
+     *
+     * @param string The string which should have capitalize
+     * @return The Capitalized string
+     */
+    public static String capitalize(String string) {
+        return WordUtils.capitalize(string);
+    }
```

Comment:
What's the point of this method if it's just calling one that looks exactly the same?